app/javascript/components/projects/ProjectsTable.vue

```diff
@@ -110,8 +146,25 @@ export default {
     getProjectAction: function (project) {
       return "/projects/" + project.id;
     },
+    destroyAction: function (project) {
+      return `/projects/${project.id}`;
```

Comment:
Map this string type (``) to other functions

Reply:
https://github.com/mitre/vulcan/pull/386/commits/56dffa21e82963ad5974b300c6f5a7831aebc777